PocketKit/Sources/PocketGraphTestMocks/UnleashAssignment+Mock.graphql.swift

```diff
@@ -0,0 +1,33 @@
+// @generated
+// This file was automatically generated and should not be edited.
+
+import ApolloTestSupport
+import PocketGraph
+
+public class UnleashAssignment: MockObject {
+  public static let objectType: Object = PocketGraph.Objects.UnleashAssignment
+  public static let _mockFields = MockFields()
+  public typealias MockValueCollectionType = Array<Mock<UnleashAssignment>>
+
+  public struct MockFields {
+    @Field<Bool>("assigned") public var assigned
+    @Field<String>("name") public var name
+    @Field<String?>("payload") public var payload
```

Comment:
Note there is a bug in Apollo GraphQL: https://github.com/apollographql/apollo-ios/issues/2977